public/data/ShlokMohanty.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Shlok Mohanty",
+  "bio": "Member of EddieHub, Machine Learning ,Data Science , Full stack Development enthusiast",
```

Comment:
Check the comma after `Machine Learning` and `Data Science`

Reply:
Nice catch! Something like this would be better, I guess.
```suggestion
  "bio": "Member of EddieHub, Machine Learning, Data Science, and Full-stack developer enthusiast",
```

Reply:
@Panquesito7 should it be developer or development  ?

Reply:
@Panquesito7 are the changes now resolved?